src/number_theory/lucas_lehmer.lean

```diff
@@ -438,7 +438,9 @@ begin
 end
 
 -- Here we calculate the residue, very inefficiently, using `dec_trivial`. We can do much better.
-example : (mersenne 5).prime := lucas_lehmer_sufficiency 5 (by norm_num) dec_trivial
+-- TODO: norm_num claims the second goal is false
+-- example : (mersenne 5).prime := lucas_lehmer_sufficiency 5 (by norm_num)
+--   (by norm_num [lucas_lehmer_residue, lucas_lehmer_test, s_zmod])
```

Comment:
This is really strange.  The second argument to `lucas_lehmer_sufficiency` used to be provable by `dec_trivial`, but now `norm_num` claims it is false.

Reply:
Does `dec_trivial` work again here in 3.30.0c?

Reply:
`dec_trivial` works again, but `norm_num` still claims this is false.

Reply:
Ah, I believe `norm_num` can't prove it because it's in `zmod`.